### ISSUE TITLE
You account might got banned.

### ISSUE THREAD

**Issue by kingkai808** [pokesniperandroid/PokeSniper-Android#59]
My account is fine. You replaced the unknown error exception with your account might got(*be) banned. 
It always happens when attempting to catch Pokemon. Error. I have auto transfer on and rasperberry off.

**Comment by kingkai808** [comment]
Too bad you cant just make a version 1.2 without ads. Was the best version. 1.4 has too many errors while trying to catch pokemon

**Comment by cire27rn** [comment]
I did not have any issues on 1.4. Only razzberry.

**Comment by kingkai808** [comment]
Are you using donated version or free? I'm on donated version.

**Comment by cire27rn** [comment]
Im using the donated version. Sorry i do not the issues on a donated version

**Comment by getmoney808** [comment]
King kai can u help with the home corrrdinates

**Comment by kingkai808** [comment]
Has to be exact coordinates of your last login or close to it (100meters)

**Comment by snowpool5609** [comment]
I'm having the same problem. Have you found an solution yet or no?

**Comment by getmoney808** [comment]
Yes ur cordinates has to be where u logged in last

**Comment by snowpool5609** [comment]
I am using that. Still not working

**Comment by kingkai808** [comment]
dont worry if you're account was banned it would not let you log into the game at all saying failed to get game data.
if it says your account may be banned over and over after trying to log in, you may have been ip soft/perm banned, if so then change your ip. or load the real game on your phone and check. if it says unable to authenticate then it means its ip banned.

**Comment by parind** [comment]
The coordinates in the settings should not havw any spaces between them. Plus check ur Account if the Pokemons are not 250/250. And try the 1.4.1. Works for me !

**Comment by snowpool5609** [comment]
Nope I'm not banned. I can catch Pokemon and do everything then go sit and get my current coordinates and no matter what I do the snipe thing always says I might have gotten banned when I haven't

**Comment by snowpool5609** [comment]
Also I have the lastest version and my box isn't full

**Comment by snowpool5609** [comment]
And I did the no spaces thing

**Comment by parind** [comment]
And ya there seems to be a problem in the app. When Pokeball reaches 0. For the first time it may pass on "Your account seems to be Banned" and if the second time it passes out "null". It clearly determines that there are no Pokeballs left.

**Comment by kingkai808** [comment]
delete and reinstall 1.4.1 
app is working perfectly right now for me

**Comment by snowpool5609** [comment]
I have like 15 pokeballs left

**Comment by snowpool5609** [comment]
I guess I can try deleting and reinstalling it.

**Comment by JrMalhar** [comment]
Hello all. Plz help. 
I have logged in with my PTC account. But it says "Your Account Might Got Banned". I caught a pokemon with the official app and within minutes I tried pokesniper and it says the same thing. It works fine with Gmail account but why not with my PTC? Where have I gone wrong? I'm not even soft banned. I tried both versions 1.4.1 & 1.4. My ptc account has 50great balls and 25 ultra balls but 0 normal pokeballs. Is it because of this? I'm using correct pokemon Co ordinates and also correct home Co ordinates. Plz help
